src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1902,10 +1925,16 @@ public CitizenNameFile getCitizenNameFile()
     /**
      * Check if we need to update the view's chunk ticket info
      *
-     * @return
+     * @return true if dirty.
      */
     public boolean isTicketedChunksDirty()
     {
         return ticketedChunksDirty;
     }
+
+    @Override
+    public short getDay()
+    {
+        return day;
+    }
```

Comment:
This might be best changed to an integer anyway.  On a server, chunk-loaded, a colony will eat through 65k days in a little under 2 years real time.  Which may sound like a lot, but given that it is intended to take several months to play a colony through, it wouldn't surprise me if some people are still playing colonies that long.  Certainly I still have some colonies that are that old, although they haven't been running continuously and so wouldn't have counted that far yet.

Reply:
Actually, 11 months, when using a signed short...